<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Local_and_Download</class>
 <widget class="QWidget" name="Local_and_Download">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QTableWidget" name="tableWidget">
       <property name="minimumSize">
        <size>
         <width>890</width>
         <height>500</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="btn_LocalMusic">
           <property name="minimumSize">
            <size>
             <width>95</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>95</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>本地音乐</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>15</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_Download">
           <property name="minimumSize">
            <size>
             <width>95</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>95</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>下载管理</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>18</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="lab_MusicSum">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>本地共有 0 首歌曲</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="line_search">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>29</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>29</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>搜索本地音乐</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_openFile">
             <property name="minimumSize">
              <size>
               <width>95</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>95</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>打开目录</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
